### PR TITLE
chore: Add download links

### PR DESCRIPTION
**Pull request by felixrieseberg**
With all the new architectures, download links aren't as easy to scan anymore. This PR adds download links to the README. We'll have to manually update them, but they'll make it easier to just download the right thing.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,11 +10,95 @@ save your Fiddle either as a GitHub Gist or to a local folder. Once published
...
+    <td>
+      <img src="./.github/images/macos.png" width="24"><br />
+      macOS
+    </td>
+    <td>
+      <span>Intel Processor</span>
+      <a href="https://github.com/electron/fiddle/releases/download/v0.19.0/Electron.Fiddle-darwin-arm64-0.19.0.zip">
+        📦 Standalone Zip
+      </a><br />
+      <span>Apple M1 Processor</span>
```
Comment (erickzhao) [inline README.md:54]: nit: Based on apple's naming conventions, I'm guessing `M1` will be outdated by next year. 😅 

```suggestion
      <span>Apple Silicon Processor</span>
```

File: README.md [hunk 2]
```diff
@@ -10,11 +10,95 @@ save your Fiddle either as a GitHub Gist or to a local folder. Once published
...
 address bar.
 
-<h3 align="center">
-  <a href="https://github.com/electron/fiddle/releases/latest">
-  Download Electron Fiddle for macOS, Windows, and Linux
-  </a>
-</h3>
+# Download
+
+<table class="is-fullwidth">
```
Comment (erickzhao) [inline README.md:15]: Is this CSS class doing anything?

File: README.md [hunk 3]
```diff
@@ -10,11 +10,95 @@ save your Fiddle either as a GitHub Gist or to a local folder. Once published
...
+</thead>
```
Comment (erickzhao) [inline README.md:16]: Stray tag
```suggestion
```

File: README.md (hunk 2 above)
Comment (felixrieseberg) [reply README.md:15]: According to some obscure internet thingies it might, but I haven't seen it in action just yet

**Review by erickzhao — APPROVED** [review]

**Pull request merged**